common/src/main/java/dev/lavalink/youtube/track/YoutubeAudioTrack.java

```diff
@@ -66,6 +69,10 @@ public void process(LocalAudioTrackExecutor localExecutor) throws Exception {
     }
 
     try (HttpInterface httpInterface = sourceManager.getInterface()) {
+      JsonObject userData = JsonParser.object().from(getUserData().toString());
```

Comment:
you should ensure to catch the parse error and ignore it if this does not return json